API.md

```diff
@@ -1694,3 +1762,71 @@ Implements functions to deal with a Tag object.
 
 ### tag.json() ⇒ <code>Any</code>
 **Kind**: instance method of [<code>Tag</code>](#Tag)  
+<a name="assignUidToParameterSchemas"></a>
+
+## assignUidToParameterSchemas(doc)
+Assign parameter keys as uid for the parameter schema.
+
+**Kind**: global function  
+
+| Param | Type |
+| --- | --- |
+| doc | [<code>AsyncAPIDocument</code>](#AsyncAPIDocument) | 
+
+<a name="assignUidToComponentSchemas"></a>
+
+## assignUidToComponentSchemas(doc)
+Assign uid to component schemas.
+
+**Kind**: global function  
+
+| Param | Type |
+| --- | --- |
+| doc | [<code>AsyncAPIDocument</code>](#AsyncAPIDocument) | 
+
+<a name="assignNameToAnonymousMessages"></a>
+
+## assignNameToAnonymousMessages(doc)
+Assign anonymous names to nameless messages.
+
+**Kind**: global function  
+
+| Param | Type |
+| --- | --- |
+| doc | [<code>AsyncAPIDocument</code>](#AsyncAPIDocument) | 
+
+<a name="recursiveSchema"></a>
+
+## recursiveSchema(schema, callback(schema))
+Recursively go through each schema and execute callback.
+
+**Kind**: global function  
+
+| Param | Type | Description |
+| --- | --- | --- |
+| schema | [<code>Schema</code>](#Schema) | found. |
+| callback(schema) | <code>function</code> | the function that is called foreach schema found.         schema {Schema}: the found schema. |
+
+<a name="schemaDocument"></a>
+
+## schemaDocument(doc, callback(schema))
+Go through each channel and for each parameter, and message payload and headers recursively call the callback for each schema.
+
+**Kind**: global function  
+
+| Param | Type | Description |
+| --- | --- | --- |
+| doc | [<code>AsyncAPIDocument</code>](#AsyncAPIDocument) |  |
+| callback(schema) | <code>function</code> | the function that is called foreach schema found.         schema {Schema}: the found schema. |
+
+<a name="assignIdToAnonymousSchemas"></a>
+
+## assignIdToAnonymousSchemas(doc)
```

Comment:
I see it is added here as it was not added in previous PRs. This will happen constantly.
What do you think if in a separate PR will add to release workflow a step with docs generation, so they get into npm and also get into the repository with package.json bump pr https://github.com/asyncapi/parser-js/blob/master/.github/workflows/release.yml#L41

Reply:
YES!